server/routes/api/landing_page.py

```diff
@@ -41,20 +41,23 @@
 OVERVIEW = 'Overview'
 
 
-def get_landing_page_data(dcid, new_stat_vars):
-    return get_landing_page_data_helper(dcid, '^'.join(new_stat_vars))
+def get_landing_page_data(dcid, new_stat_vars, category):
+    return get_landing_page_data_helper(
+        dcid,
+        '^'.join(new_stat_vars),
+        category,
+    )
 
 
 @cache.memoize(timeout=3600 * 24)  # Cache for one day.
-def get_landing_page_data_helper(dcid, stat_vars_string):
-    data = {'place': dcid}
+def get_landing_page_data_helper(dcid, stat_vars_string, category):
+    stat_var_condition = ''
     if stat_vars_string:
-        data['newStatVars'] = stat_vars_string.split('^')
-    response = dc_service.fetch_data('/landing-page',
-                                     data,
-                                     compress=False,
-                                     post=True,
-                                     has_payload=False)
+        stat_var_condition = "&new_stat_vars={}".format(
+            stat_vars_string.split('^'))
+    req_url = '/v1/internal/page/place/{}?category={}'.format(
```

Comment:
Can you add a wrapper function in https://github.com/datacommonsorg/website/blob/master/server/services/datacommons.py#L163-L169.

We try to put all the API access function in datacommons.py

Reply:
Also try to use the POST endpoint https://github.com/datacommonsorg/mixer/pull/931.

With both changes, you can remove the get_landing_page_data_helper() which was used for caching.

Reply:
Done